cli/internal/context/context.go

```diff
@@ -177,18 +144,19 @@ func WithGraph(rootpath string, config *config.Config) Option {
 			c.Lockfile = lockfile
 		}
 
-		if c.resolveWorkspaceRootDeps(rootPackageJSON) != nil {
-			return err
+		if err := c.resolveWorkspaceRootDeps(config.RootPackageJSON); err != nil {
+			// TODO(Gaspar) was this the intended return error?
+			return fmt.Errorf("could not resolve workspaces: %w", err)
 		}
```

Comment:
@jaredpalmer , previously this was returning the previous `err` since the error from `c.resolveWorkspaceRootDeps(rootPackageJSON)` was not assigned. I assume that was unintentional.

Reply:
Good catch, I'm going to assume that your change is in fact what we want.